s3tests_boto3/functional/test_s3.py

```diff
@@ -340,6 +474,40 @@ def test_bucket_list_delimiter_unreadable():
     eq(keys, key_names)
     eq(prefixes, [])
 
+@attr(resource='bucket')
+@attr(method='get')
+@attr(assertion='non-printable delimiter can be specified')
+@attr('list-objects-v2')
+def test_bucket_listv2_delimiter_unreadable():
+    key_names=['bar', 'baz', 'cab', 'foo']
+    bucket_name = _create_objects(keys=key_names)
+    client = get_client()
+
+    response = client.list_objects_v2(Bucket=bucket_name, Delimiter='\x0a')
+    eq(response['Delimiter'], '\x0a')
+
+    keys = _get_keys(response)
+    prefixes = _get_prefixes(response)
+    eq(keys, key_names)
+    eq(prefixes, [])
+
+@attr(resource='bucket')
+@attr(method='get')
+@attr(assertion='non-printable delimiter can be specified')
+@attr('list-objects-v2')
+def test_bucketv2_list_delimiter_unreadable():
```

Comment:
@albIN7 this test has been written twice. Could you please delete the duplicate.

Reply:
Deleted.